src/app/app.module.ts

```diff
@@ -29,8 +29,9 @@ import { SharedModule } from './modules/shared/shared.module';
 import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
 import { FooterComponent } from './components/footer/footer.component';
 import { HttpErrorInterceptor } from './interceptors/http.interceptor';
-import { SentryErrorHandler } from './error-handler/sentry.error-handler';
 import { errorHandlerFactory } from './error-handler/error-handler-factory';
+import { AlertModule } from 'ngx-bootstrap/alert';
+import { AlertComponent } from './components/alert/alert.component';
```

Comment:
Relative path, please change it to an absolute one

Reply:
I think since all modules and services right now use relative path instead of absolute paths we should keep this for now relative. We should discuss this upcoming week what kind of imports we want to use. Within this Github issue I found a cool way to use absolute paths: https://github.com/angular/angular-cli/issues/1465#issuecomment-236054952